src/reformatters/common/region_job.py

```diff
@@ -573,6 +576,7 @@ def _download_processing_group(
         """
 
         def _call_download_file(coord: SOURCE_FILE_COORD) -> SOURCE_FILE_COORD:
+            # TODO: move exception handling to the download_file method
```

Comment:
@aldenks My memory could be wrong on this, but I believe we discussed making this change in a follow up PR. I'm happy to roll it into this one though.